pkg/cli/upgradeassistant/cmd/migrate/1100.go

```diff
@@ -179,13 +180,13 @@ func changePolicyCollectionName() error {
 		return err
 	}
 
-	var ois []interface{}
-	for _, obj := range res {
-		ois = append(ois, obj)
-	}
-	if _, err = newPolicyMetaColl().InsertMany(ctx, ois); err != nil {
-		log.Errorf("Failed to InsertMany policyMetas, err: %s", err)
-		return err
+	for _, v := range res {
+		query := bson.M{"resource": v.Resource}
```

Comment:
Is it possible that there will be multiple documents in `policy_meta` with the same `resource`?

Reply:
no